src/index.js

```diff
@@ -329,19 +371,18 @@ module.exports = class Face3D {
         if (!this.scene)
             return
 
-        // Rotate camera
-        var velocityDelta = Math.min(1, delta * 6)
-        this.cameraContainer.rotation.x += (this.cameraRotationTarget.x - this.cameraContainer.rotation.x) * velocityDelta
-        this.cameraContainer.rotation.y += (this.cameraRotationTarget.y - this.cameraContainer.rotation.y) * velocityDelta
-        this.cameraContainer.rotation.z += (this.cameraRotationTarget.z - this.cameraContainer.rotation.z) * velocityDelta
-        this.camera.position.z += (this.desiredCameraZ - this.camera.position.z) * velocityDelta
-
         // Progres animations
         this.animation && this.animation.update(delta)
 
         // Do render
+        this.renderer.gammaOutput = true;
```

Comment:
These should not be re-set on every frame, rather set them above when the renderer is created...